### ISSUE TITLE
Add support for custom OSRS tooltips

### ISSUE THREAD

**Issue by osrsbox** [osrsbox/osrsbox-tooltips#10]
Currently the OSRS tooltip library only provides two specific formats. This is determined by if the item is equipable or not. The image below displays the default tooltip format for equipable items (left) and non-equipable items (right).

![tooltip-types](https://user-images.githubusercontent.com/26445681/48925270-d10ebd80-ef26-11e8-8e83-1b397c57766e.png)

It would be beneficial to add support to request tooltips with specific data, or exclude specific data. For example, leave out the highalch and lowalch values. It might be better to have a choice of specific profiles (this would be easier to implement). For example:
1. Full: Show icon, name and all metadata properties
1. Short: Show icon and name only
1. Bonuses: Only show icon, name and  item bonuses

**Comment by jburleigh1** [comment]
Hey!

I noticed you implemented this in 1.0.2 :) very nice.

One small bug I've noticed, if you have multiple tooltips on the same page, if any of them have the same IDs, only the first one will work. Tested with the following code. 

```
<span class="osrstooltip" id="11802" data-type="half" title="Please wait...">Armadyl Godsword</span>
<span class="osrstooltip" id="11802" data-type="bonuses" title="Please wait ...">Armadyl Godsword</span>
```

This causes the following error in console:
`Uncaught TypeError: j.indexOf is not a function`

Hopefully it's not an isolated issue, let me know if you can't reproduce!

**Comment by jburleigh1** [comment]
Did some further testing, it seems this line:

`Uncaught TypeError: j.indexOf is not a function` is unrelated, this error seems to be specific to my WordPress site.

The bug still exists however in an isolated test environment I setup, using html5 boilerplate.

**Comment by osrsbox** [comment]
Thanks @jburleigh1 for the feedback. I opened another issue for the problem when using the same item ID twice on the same page, and will look into the problem.

What do you think about the solution I implemented in version 1.0.2? I guess you saw the code which allows a `full`, `half`, `bonuses` and `short` layouts. Is there another layout you would be interested in? If not, I might close the issue.

**Comment by jburleigh1** [comment]
great, I think what you've done with the different layout options is perfect, I'd say you're good to close it out.

**Comment by osrsbox** [comment]
Issue closed with version 1.0.2
